dangerfile.ts

```diff
@@ -0,0 +1,8 @@
+import { danger, warn } from 'danger'
+
+// Check for a CHANGELOG entry
+const hasChangelog = danger.git.modified_files.some(f => f === 'CHANGELOG.md')
+
+if (!hasChangelog) {
+  warn('Please add a changelog entry for your changes.')
```

Comment:
maybe something like `Ensure that CHANGELOG is not updated intentionally.` or `There are no updates provided to CHANGELOG. Ensure there are no publicly visible changes introduced by this PR.` The benefit is that we won't blindly suggest to update CHANGELOG for those cases where it might not be necessary